### PR TITLE
Update docs to reflect Commit 7ad9104

### PR DESCRIPTION
**Pull request by ProdPreva1l**
Update docs to reflect [Commit 7ad9104](https://github.com/WiIIiam278/HuskHomes/commit/7ad91046778acc551d7b2c96f07bd08e431cabab)

### CODE REVIEW

**Comment by ProdPreva1l** [comment]
I really need to learn how to make git not do that

**Review by WiIIiam278 — CHANGES_REQUESTED** [review]
Can you revert your changes to `Settings.java`?

**Comment by ProdPreva1l** [comment]
I could swear i did, someone needs to send me a git master class lesson

**Comment by ProdPreva1l** [comment]
Done

File: docs/Database.md [hunk 1]
```diff
@@ -25,15 +26,15 @@ database:
   type: SQLITE
   mysql:
     credentials:
-      # Specify credentials here if you are using MYSQL or MARIADB as your database type
+      # Specify credentials here if you are using MYSQL, MARIADB, or POSTGRESQ as your database type
```
Comment (PseudoResonance) [inline docs/Database.md:29]: There's a typo here in POSTGRESQL

**Comment by PseudoResonance** [comment]
I just realized I forgot to update the documentation in my change and came here to check, when I saw this. Thanks!

**Review by WiIIiam278 — APPROVED** [review]

**Comment by WiIIiam278** [comment]
Something that slipped my mind. Thanks!

**Pull request merged**